packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/Chooser.js

```diff
@@ -42,22 +41,14 @@ function Chooser ({
   selectedChoiceIds = [],
   task
 }) {
-  const [filterDropOpen, setFilterDropOpen] = useState(false)
-
-  function clearFilters() {
-    handleFilter()
-  }
-
-  function handleFilterDropClose () {
-    setFilterDropOpen(false)
-  }
-
-  function handleFilterDropOpen () {
-    setFilterDropOpen(true)
-  }
+  const [filterOpen, setFilterOpen] = useState(false)
```

Comment:
I think adding Grommet's AnnounceContext to announce how many choices ("Showing x of y choices") after applying a filter is straightforward:
```
const announce = useContext(AnnounceContext)

useEffect(function announceFilteredChoiceCount () {
  announce(`Showing ${filteredChoiceIds.length} of ${task.choices.size} choices`, 'polite', 1000)
}, [filteredChoiceIds?.length, task.choices.size, announce])
```
Not sure if it would be helpful or unhelpful noise? Once a user tabs to the menu item list similar information ("item x of y") is announced, so this might not be a meaningful change.

Reply:
I don't have a definitive opinion on this, but considering it's few lines of code and you thought it worth investigating, happy to leave it in for now unless a volunteer reports difficulty using accessibility tools and the survey task.